csfieldguide/static/interactives/distance-from-stereo-images/README.md

```diff
@@ -0,0 +1,6 @@
+# Distance From Stereo Images Interactive
+
+**Original Author:** Jackie Qiu
+
+This interactive is created to show obtaining distance information from two stereo images.
+The link to the diagrams in the Depth section can be found [here](https://drive.google.com/file/d/1nAVHKnhzE_Mb8bM2hJ22p_45v8xdnY1O/view?usp=sharing).
```

Comment:
Could you please include this file in the interactive folder instead, in case your Google Drive ever goes offline.

Reply:
Added a .drawio file that can be imported in diagrams.net, and updated README accordingly.